### PR TITLE
Integrate React Quill

### PR DESCRIPTION
**Pull request by AntonRozhanskii**
This has been made to allow policy content formatting,
such as headings, lists, etc.

### CODE REVIEW

File: src/views/commonPage/DisplayPolicy.jsx [hunk 1]
```diff
@@ -1,154 +1,129 @@
...
+    componentDidMount() {
+        Axios.get("/reviewPolicy", {
+            params: {
+                company_name: localStorage.getItem("session_name"),
+                policy_name: localStorage.getItem('reviewPolicy')
+            }
+        })
+            .then(response => {
+                console.log("response:")
+                console.log(response)
```
Comment (AntonRozhanskii) [inline src/views/commonPage/DisplayPolicy.jsx]:
```suggestion
```

**Comment by AntonRozhanskii** [comment]
Note, that within [src/views/commonPage/DisplayPolicy.jsx](https://github.com/std702-itpm/itPolicyManager-app-react/pull/23/files#diff-7db9ddb92f5401368799a2fae6987208) are mostly formatting changes.
The meaningful changes in this file are `<PolicyEditor/>` addition, and content changes handling function.

**Review by AntonRozhanskii — COMMENTED** [review]
Forgot to remove dubugging code)

File: src/views/commonPage/printPreview.jsx [hunk 2]
```diff
@@ -108,10 +78,9 @@ export default class printPreview extends Component {
                         <p>{localStorage.getItem("session_name")}</p>
                         <p>Subscribed Date:{this.state.policy.date_subscribed}</p>
                         <p>Version: {this.state.policy.version}</p>
-                        <p>Review Date:{this.state.review_date}
-                        </p>
+                        <p>Review Date: {this.getCurrentDate()}</p>
                         <p>Approval Date: {this.state.approval_date}</p>
-                        <p>Accountable Person: Kristof C </p>
+                        <p>Accountable Person: Kristof C (hardcoded) </p>
```
Comment (ppuarat) [inline src/views/commonPage/printPreview.jsx]:
```suggestion
                        <p>Accountable Person: Kristof C</p>
```

**Review by ppuarat — CHANGES_REQUESTED** [review]

**Review by ppuarat — APPROVED** [review]

**Pull request merged**